pkg/cmd/cli/describe/printer.go

```diff
@@ -33,7 +33,7 @@ var (
 	imageStreamImageColumns = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
 	imageStreamColumns      = []string{"NAME", "DOCKER REPO", "TAGS", "UPDATED"}
 	projectColumns          = []string{"NAME", "DISPLAY NAME", "STATUS"}
-	routeColumns            = []string{"NAME", "HOST/PORT", "PATH", "SERVICE", "LABELS"}
+	routeColumns            = []string{"NAME", "HOST/PORT", "PATH", "SERVICE", "LABELS", "TLS"}
```

Comment:
I thought that might be a little wordy but have no preference.  

@fabianofranz - what do you think?  In reality, this is more of the "non-secure route type" since it could be a pass through route as well rather than just TLS termination but that is confusing to end users.